config/deploy/production.rb

```diff
@@ -33,7 +33,7 @@
 # Feel free to add new variables to customise your setup.
 
 set :rbenv_type, :user
-set :rbenv_ruby, '2.3.3'
+set :rbenv_ruby, '2.4.1'
```

Comment:
I checked the server and 2.4.1 is already installed in rbenv, so I'm hoping that this line will automatically switch the app over to use it.